content/cloud-computing/concepts/types-of-cloud/types-of-cloud.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: 'Types of Cloud'
+Description: 'There are three types of cloud computing implementations.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Networking'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are three **types** of cloud computing implementations:
+
+## Public Cloud
+
+Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.
```

Comment:
I prefer lists as bullet points rather than listing clauses in a sentence, but up to you.

Reply:
Good spot!

Reply:
```suggestion
Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include the following:

- [Amazon Web Services (AWS)](https://aws.amazon.com/)
- [Google Cloud Platform (GCP)](https://cloud.google.com/gcp)
- [Microsoft Azure](https://azure.microsoft.com/en-us/)
```